dfm/data/load.py

```diff
@@ -187,3 +204,37 @@ def load_dfm_dataset(dataset: str, **kwargs) -> Dataset:
         raise ValueError(
             f"invalid dataset. Valid datasets include {dataset_loaders.keys()}"
         )
+
+
+def load_multiple_dfm_datasets(datasets: List[str], **kwargs) -> Dataset:
```

Comment:
Missing docstring

Reply:
Well the entire load script is lacking a lot. I feel that those who made it should get the chance to clean it up a bit themselves.

Reply:
Sure, but since this PR is merging in these functions, it should probably be fixed as otherwise nobody will ever do it 😛

Reply:
Added docstrings :)